crates/viewer/re_space_view_spatial/src/visualizers/boxes3d.rs

```diff
@@ -43,133 +35,37 @@ impl Default for Boxes3DVisualizer {
-                        let box3d = line_batch
-                            .add_box_outline_from_transform(world_from_instance)
-                            .color(color)
-                            .radius(radius)
-                            .picking_instance_id(PickingLayerInstanceId(instance_index as _));
-
-                        if let Some(outline_mask_ids) =
-                            ent_context.highlight.instances.get(&instance)
-                        {
-                            box3d.outline_mask_ids(*outline_mask_ids);
-                        }
-                    }
-                    FillMode::Solid => {
-                        let Some(solid_mesh) =
-                            ctx.viewer_ctx.cache.entry(|c: &mut proc_mesh::SolidCache| {
-                                c.entry(proc_mesh_key, render_ctx)
-                            })
-                        else {
-                            return Err(SpaceViewSystemExecutionError::DrawDataCreationError(
-                                "Failed to allocate solid mesh".into(),
-                            ));
-                        };
-
-                        mesh_instances.push(MeshInstance {
-                            gpu_mesh: solid_mesh.gpu_mesh,
-                            mesh: None,
-                            world_from_mesh: world_from_instance,
-                            outline_mask_ids: ent_context.highlight.index_outline_mask(instance),
-                            picking_layer_id: picking_layer_id_from_instance_path_hash(
-                                InstancePathHash::instance(entity_path, instance),
-                            ),
-                            additive_tint: color,
-                        });
-                    }
-                }
-            }
-
-            self.0
-                .bounding_boxes
-                .push((entity_path.hash(), world_space_bounding_box));
-
-            self.0.ui_labels.extend(process_labels_3d(
-                LabeledBatch {
-                    entity_path,
-                    num_instances,
-                    overall_position: world_space_bounding_box.center(),
-                    instance_positions: ent_context
-                        .transform_info
-                        .clamped_reference_from_instances()
-                        .map(|t| t.translation.into()),
-                    labels: &data.labels,
-                    colors: &colors,
-                    show_labels: data.show_labels.unwrap_or_else(|| self.fallback_for(ctx)),
-                    annotation_infos: &annotation_infos,
+        let proc_mesh_key = proc_mesh::ProcMeshKey::Cube;
+
+        // `ProcMeshKey::Cube` is scaled to side length of 1, i.e. a half-size of 0.5.
+        // Therefore, to make scaling by half_size work out to the correct result,
+        // apply a factor of 2.
+        // TODO(kpreid): Is there any non-historical reason to keep that.
```

Comment:
no, not that I can think of. Can be cleaned up at will! no

Reply:
I'll consider that for a future PR, and not make this PR hairier by changing the size here.